aspnet/tutorials/first-web-api.rst

```diff
@@ -115,9 +111,11 @@ In the ``ConfigureServices`` method, add the highlighted code:
 
 .. literalinclude:: first-web-api/sample/src/TodoApi/Startup.cs
   :language: c#
-  :lines: 13-23
-  :emphasize-lines: 9-10
+  :start-after: #region snippet_AddSingleton
+  :end-before: #endregion
+  :emphasize-lines: 6
   :dedent: 8
+  :linenos:
```

Comment:
Did you intend to leave line numbers in?
Also, the actual code created in VS includes an App Insights line that doesn't show in this snippet.